src/app/ReadHandler.cpp

```diff
@@ -202,6 +202,7 @@ CHIP_ERROR ReadHandler::SendReportData(System::PacketBufferHandle && aPayload, b
     if (IsPriming() || IsChunkedReport())
     {
         mSessionHandle.Grab(mpExchangeCtx->GetSessionHandle());
+        mpExchangeCtx->SetResponseTimeout(kImMessageTimeout);
```

Comment:
Why not just put this after the whole if/else instead of in each branch separately?

Reply:
Done， thanks